### ISSUE TITLE
Possible bug if ran in incognito mode.

### ISSUE THREAD

**Issue by DarkAlchy** [privacypass/challenge-bypass-extension#181]
**Describe the bug**
A clear and concise description of the bug.
I should not be asked to do a captcha after the first time with this addon running and I don't as long as I am not in incognito mode.  Even though I said to allow this to run is incognito mode, and it is there showing me the passes I have, it appears it is being ignored.

**To Reproduce**
1. Go to '...'  https://www.gg2u.org
2. Click on '....' Log in
3. Scroll down to '....' Nothing

**Expected behavior**
It should just ask me for my login credentials.

**System (please complete the following information):**
 - OS: [e.g. iOS] Windows 10
 - Cloudflare tokens or hCaptcha tokens? CF tokens is 57 and hCaptcha is 4 but just fell to 1.
 - Browser [e.g. chrome, firefox] Chrome
 - Browser Version [e.g. 79, 80, ] 83.0.4103.97 (Official Build) (64-bit)
 - Privacy Pass Version [e.g. 2.0.4, 2.0.5 ] 2.0.6
 - Did you install Privacy Pass from this repository or from the browser store? Browser store.

**Additional context**
This is only has a problem when in Incognito mode.

**Comment by alxdavids** [comment]
Thanks, this is very helpful, let me look into it.

**Comment by alxdavids** [comment]
I think I'm not seeing the same behaviour. When I have CF passes and navigate to gg2u.org (and press the log in button) I am able to access both pages. I can see that tokens are being spent to bypass the challenge page on gg2u.org because I see the challenge page for a second before I'm redirected.

I presume what you mean is that you're still seeing the challenge page with the hCaptcha, even when you have tokens?

**Comment by DarkAlchy** [comment]
Yes, I have the tokens there (57 and now 1 as I used the other three when I tested one time in non incognito mode) but I am still sent to the page with the captcha (by cloudfare).  I get this no matter what if in incognito mode - https://i.imgur.com/DwV42gJ.png  If I am not in incognito mode I don't get asked.

**Comment by ValZapod** [comment]
You must permit to use the extension in Incognito mode here chrome://extensions/?id=ajhmfdgkijocedmfjonnpjfojldioehi

**Comment by DarkAlchy** [comment]
> 
> 
> You must permit to use the extension in Incognito mode here chrome://extensions/?id=ajhmfdgkijocedmfjonnpjfojldioehi

That goes without saying, or at least it should.

**Comment by migueldemoura** [comment]
I'm unable to reproduce the issue with Chrome `Version 101.0.4951.64 (Official Build) (x86_64)` in Incognito mode, feel free to reopen if the issue persists!
